ansible/galaxy.yml

```diff
@@ -26,7 +26,8 @@ tags:
 # collection label 'namespace.name'. The value is a version range
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
-dependencies: {}
+dependencies:
+  "kubernetes.core": ">=2.0.0"
```

Comment:
Do you not want to pin an exact version & bump w/renovate?

Reply:
I'm of two minds here.  While it's useful for us here to pin the version, it's not how you would typically releases these.  I would like to use it in my repo as well, and I have other roles/collections that might then conflict with a pinned version.